### DISCUSSION TITLE
HomeAssistant OS and plex media server

### DISCUSSION THREAD

**Discussion by norbertjoni** [alexbelgium/hassio-addons#740] [Q&A]
I am thinking to leave proxmox and install HA OS to my intel setup. However, I need plex, radarr, sonarr, and torrent too. 

Does anybody use HA OS with these add-ons? It's stable? Plex can use igpu for transcoding too? 

Or should I stay with proxmox?

**Comment by pbvdven** [comment]
I use all of the mentioned works well only i have plex running in docker not as ha addon so i can use my nvidia gpu. Ive installed ha supervised like this https://community.home-assistant.io/t/installing-home-assistant-supervised-on-debian-11/200253 ha os wont let u change the os. i needed to install gpu drivers and nvidia container toolkit so thats why i installed ha supervised. Instead of os.
